eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -6,249 +37,195 @@ jobs:
   condition: and(succeededOrFailed(), ne(variables['Skip.Test'], 'true'))
   strategy:
     matrix:
-      # Build with no dependencies at all (No samples) - PRECONDITIONS ON
-      # WARNINGS_AS_ERRORS          ON
-      # TRANSPORT_CURL              OFF
-      # UNIT_TESTING                OFF
-      # UNIT_TESTING_MOCKS          OFF
-      # TRANSPORT_PAHO              OFF
-      # PRECONDITIONS               ON
-      # LOGGING                     OFF
-      # AZ_PLATFORM_IMPL            AZ_PLATFORM_IMPL_NONE
-      Linux_ARM:
+      LinuxARM_Release_MapFiles_Logging:
         OSVmImage: 'ubuntu-18.04'
         vcpkg.deps: ''
+        VCPKG_DEFAULT_TRIPLET: 'x64-linux'
         AptDependencies: 'gcc-arm-none-eabi'
-        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake'
+        build.args: '-DCMAKE_TOOLCHAIN_FILE=cmake-modules/gcc-arm-toolchain.cmake -DPRECONDITIONS=OFF'
         AZ_SDK_C_NO_SAMPLES: 'true'
+        PublishMapFiles: 'true'
+        MapFileArtifactSuffix: 'lnx-arm-rel-noprc-log'
```

Comment:
Nice.

Reply:
This map file artifact is not getting published. Is something wrong here? We didn't have it before, so feel free to remove it if you can't root cause the issue.
https://dev.azure.com/azure-sdk/public/_build/results?buildId=544516&view=artifacts&type=publishedArtifacts
![image](https://user-images.githubusercontent.com/6527137/93660881-38550600-fa08-11ea-8735-8594959b5828.png)